Source/WebCore/loader/cache/CachedResourceLoader.cpp

```diff
@@ -116,15 +116,27 @@ static inline ResourceErrorOr<CachedResourceHandle<T>> castCachedResourceTo(Reso
     return makeUnexpected(cachedResource.error());
 }
 
-static CachedResourceHandle<CachedResource> createResource(CachedResource::Type type, CachedResourceRequest&& request, PAL::SessionID sessionID, const CookieJar* cookieJar, const Settings& settings)
+static ScriptRequiresTelemetry scriptRequiresTelemetry(const Document* document, const ResourceRequest& request)
+{
+    if (UNLIKELY(!document))
+        return ScriptRequiresTelemetry::No;
+
+    RefPtr page = document->protectedPage();
```

Comment:
nit - unneeded reference churn
```suggestion
    RefPtr page = document->page();
```

Reply:
Fixed!